cmd/serve/docker/volume.go

```diff
@@ -270,7 +270,7 @@ func (vol *Volume) mount(id string) error {
 		return errors.New("volume filesystem is not ready")
 	}
 
-	if err := vol.mnt.Mount(); err != nil {
+	if _, err := vol.mnt.Mount(); err != nil {
```

Comment:
Note: mounting in docker will reset Daemon to false in advance so it can ignore the `daemonized` the return.